v3/src/models/formula/formula-mathjs-scope.ts

```diff
@@ -217,6 +217,10 @@ export class FormulaMathJsScope {
     return this.context.dataSets.get(dataSetId)
   }
 
+  getDataSetByTitle(dataSetTitle: string) {
+    return Array.from(this.context.dataSets.values()).find(dataSet => dataSet._title === dataSetTitle)
```

Comment:
I would think we'd want 
```suggestion
    return Array.from(this.context.dataSets.values()).find(dataSet => dataSet.title === dataSetTitle)
```